src/controllers/ratingsController.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-constant-condition */
+import model from '../database/models';
+import fService from '../services/facilityService';
+
+const addRating = async (req, res, next) => {
+  let resp = '';
+  const facId = req.params.id;
+  try {
+    const facility = await fService.findFacilityById(facId);
+    if (!facility) {
+      resp = res.status(404).json({ message: res.__('Facility Not found!') });
+    } else {
+      req.body.facilityId = facId;
+      req.body.userId = req.userId;
+      const rating = await model.rating.create(req.body);
+      if (rating) resp = res.status(201).json({ message: res.__('rating created'), rating });
+    }
+  } catch (error) {
+    return res.status(500).json({ message: res.__('Bad request!') });
```

Comment:
This error message is misleading, bad requests are in the 4XX range. Could be, say, 'internal error.' And remember to change in multi-language support(locale files) too, if Internal error is not yet there